lib/HueSensor.js

```diff
@@ -2349,6 +2397,41 @@ HueSensor.prototype.checkVoltage = function (voltage) {
   }
 }
 
+HueSensor.prototype.checkAirQuality = function (airquality) {
+  if (this.obj.state.airquality !== airquality) {
+    this.log.debug(
+      '%s: airquality changed from %j to %j', this.name,
+      this.obj.state.airquality, airquality
+    )
+    this.obj.state.airquality = airquality
+  }
+  var hkAirQuality;
+  if (airquality == 'excellent') {
```

Comment:
I'd probably use a map, but then you would need an additional check for `null`.

Please use `===` for comparison (except `== null` to check both `null` and `undefined`).